src/badguy/bouncing_snowball.cpp

```diff
@@ -24,8 +24,10 @@
 static const float JUMPSPEED = -450;
 static const float BSNOWBALL_WALKSPEED = 80;
 
-BouncingSnowball::BouncingSnowball(const ReaderMapping& reader)
-  : BadGuy(reader, "images/creatures/bouncing_snowball/bouncing_snowball.sprite")
+BouncingSnowball::BouncingSnowball(const ReaderMapping& reader) :
+    BadGuy(reader, "images/creatures/bouncing_snowball/bouncing_snowball.sprite"),
+    m_turn_around_timer(),
+    m_turn_around_counter()
```

Comment:
Incorrect tabulation:

```suggestion
  BadGuy(reader, "images/creatures/bouncing_snowball/bouncing_snowball.sprite"),
  m_turn_around_timer(),
  m_turn_around_counter()
```